ros/iron/ubuntu/jammy/ros-core/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# This is an auto generated Dockerfile for ros:ros-core
+# generated from docker_images_ros2/create_ros_core_image.Dockerfile.em
+FROM ubuntu:jammy
+
+# setup timezone
+RUN echo 'Etc/UTC' > /etc/timezone && \
+    ln -s /usr/share/zoneinfo/Etc/UTC /etc/localtime && \
+    apt-get update && \
+    apt-get install -q -y --no-install-recommends tzdata && \
+    rm -rf /var/lib/apt/lists/*
+
+# install packages
+RUN apt-get update && apt-get install -q -y --no-install-recommends \
+    dirmngr \
+    gnupg2 \
+    && rm -rf /var/lib/apt/lists/*
+
+# setup sources.list
+RUN echo "deb http://packages.ros.org/ros2-testing/ubuntu jammy main" > /etc/apt/sources.list.d/ros2-testing.list
```

Comment:
```suggestion
RUN echo "deb http://packages.ros.org/ros2/ubuntu jammy main" > /etc/apt/sources.list.d/ros2.list
```

Reply:
Unfortunately I couldn't (as of 1hour ago) as apparently several variant packages have not been synced (e.g. simulation, desktop full etc)

Reply:
Ah true. Will wait for those to become available!

Reply:
So that should all be fixed now, but we'll need to do another sync in order to get those out.

@Yadunund We may want to consider doing a sync (not a patch release) in the next couple of weeks.  That will get `desktop-full` and `simulation` out (unblocking this), as well as getting `navigation2` out.

Reply:
@clalancette @Yadunund any chance the sync can happen sooner rather than later ? it will be pretty odd for the community to have a ROS release without desktop_full available and to wait several weeks to have docker images available

Reply:
I will also point out that we could comment out the `desktop-full` and `simulation` variants here, and release just the `core`, `base`, and `desktop` docker images.

Reply:
No blockers, just made the announcement for the sync https://discourse.ros.org/t/preparing-for-iron-sync-2023-05-31/31674

Reply:
`desktop-full` should now be available 😄